packages/core/build/webpack.config.js

```diff
@@ -12,9 +12,9 @@ module.exports = function (env, argv) {
         devServer: {
             open: openChromeBasedOnPlatform(process.platform),
             openPage: sub_path,
-            host: 'localhost.binary.sx',
-            https: true,
-            port: 443,
+            host: 'localhost',
+            https: false,
+            port: 5000,
```

Comment:
Is this intentional? If so, we should probably update `deriv-app/packages/shared/src/utils/config/config.js`

Reply:
Oh No. nice catch. I had to do this in order to be able to use SW on localhost. Reverting.